Cargo.toml

```diff
@@ -25,51 +21,26 @@ travis = []
 with-syntex = ["syntex", "pnet_macros/with-syntex"]
 
 [dependencies]
+libc = "0.2"
 ipnetwork = "0.12"
+
 pnet_base = { path = "pnet_base" }
 pnet_sys = { path = "pnet_sys" }
 pnet_datalink = { path = "pnet_datalink" }
+pnet_macros = { path = "pnet_macros", optional = true, version = ">=0.9" }
+pnet_macros_support = { path = "pnet_macros_support", version = ">=0.1" }
+pnet_macros_plugin = { path = "pnet_macros_plugin", optional = true, version = ">=0.1" }
 
-[dependencies.clippy]
-optional = true
-version = ">=0.0"
-
-[dependencies.netmap_sys]
-features = ["netmap_with_libs"]
-optional = true
-version = ">=0.0"
-
-[dependencies.libc]
-version = "0.2.*"
```

Comment:
A lot of the dependencies were very simple, just a version. I do not see any reason to have them under individual blocks like this. Just makes this file super long.

Reply:
It is mostly this way because historically I had no idea how to work Cargo 😉 

Thanks for cleaning it up.